front/src/applications/stdcm/hooks/useStdcm.ts

```diff
@@ -52,6 +52,8 @@ const useStdcm = ({
   const { getConf, getTimetableID } = useOsrdConfSelectors();
   const osrdconf = useSelector(getConf) as OsrdStdcmConfState;
   const timetableId = useSelector(getTimetableID);
+  const requestPromise = useRef<ReturnType<typeof postTimetableByIdStdcm>>();
+  const isRequestCancelled = useRef(false);
```

Comment:
Instead of having a ref, could we `try`/`catch` and check if the error is an `AbortError`?

Reply:
You're saying we should use AbortController, with signal, in a try catch, to see if we have an AbortError ? And if it's the case, return early ?

Reply:
No need for `AbortController`, I think the error should already be an `AbortError` if we call `abort()` like already done in this PR? Or is the error a completely different kind?